kolibri/core/lessons/serializers.py

```diff
@@ -136,7 +136,7 @@ def update(self, instance, validated_data):
                     collection__kind=ADHOCLEARNERSGROUP
                 ).values_list("collection__id", flat=True)
             )
-            new_group_ids = set(x.id for x in collections)
+            new_group_ids = {x.id for x in collections}
```

Comment:
TIL about [set comprehensions](https://docs.python.org/3/tutorial/datastructures.html#sets).

To be honest, I personally prefer the `set` syntax for the extra three characters `¯\_(ツ)_/¯` because this looks like a broken dict to me...

Anyways no biggie

Reply:
It's a minor performance win to use the comprehension directly.